playground/test_playground.sh

```diff
@@ -53,13 +104,13 @@ orderUid=$( curl --fail-with-body -s -X 'POST' \
   "sellAmount": "'$sellAmount'",
   "buyAmount": "'$buyAmount'",
   "validTo": '$validTo',
-  "feeAmount": "0",
+  "feeAmount": "'$feeAmount'",
```

Comment:
Due to legacy reasons this part is a bit annoying. All new orders are supposed to have `feeAmount: 0`. So in order to incorporate the fee in your order you would have to deduct the fee from your `sellAmount` (fees are always expressed in the sell token).